other-examples/aws/memcached/nodejs/src/server.js

```diff
@@ -0,0 +1,58 @@
+const express = require("express");
+const path = require("path");
+const routes = require("./api/routes");
+
+const Memcached = require('memcached');
+
+const MEMCACHED_CONFIGURATION_ENDPOINT = 'test-memcached.abc123.cfg.use1.cache.amazonaws.com:11211'; // Replace with your Memcached configuration endpoint
```

Comment:
Why is this specific to AWS? Do we only support relationships to memcached if it is hosted in AWS?

Is there anything a customer needs to do to configure their memcached instance so that it is instrumented? If so, where are the instructions for this?